luaui/Widgets/gui_build_eta.lua

```diff
@@ -175,6 +175,8 @@ local function drawEtaText(timeLeft, yoffset)
 	if timeLeft == nil then
 		etaText = etaPrefix .. "\255\1\1\255???"
 	else
+		etaPrefix = ((timeLeft>0) and etaPrefix) or " \255\255\1\1CANCELED "
```

Comment:
All text displayed to the player needs to use the I18N module, to support translations.

Reply:
Canceled is American, Cancelled is British English